app.json

```diff
@@ -6,16 +6,79 @@
   "keywords": ["commercetools", "ecommerce", "shopping"],
   "env": {
     "CTP_PROJECT_KEY": {
-      "description": "The project key for your commercetools platform project.",
-      "required": true
+      "description": "The project key for your commercetools platform project."
     },
     "CTP_CLIENT_ID": {
-      "description": "The client ID of your commercetools platform project.",
-      "required": true
+      "description": "The client ID of your commercetools platform project."
     },
     "CTP_CLIENT_SECRET": {
-      "description": "The client secret of your commercetools platform project.",
-      "required": true
+      "description": "The client secret of your commercetools platform project."
+    },
+
+    "SELECTABLE_ATTRIBUTES": {
+      "description": "The product attributes that allow to identify and select a product variant in your project. Enter the attribute keys separated by commas."
+    },
+
+    "APPLICATION_SECRET": {
+      "description": "The secret key is used to secure cryptographics functions. Anyone that can get access to the application secret will be able to generate any session they please, effectively allowing them to log in to your system as any user they please. If you deploy your application to several instances be sure to use the same key.",
+      "generator": "secret"
```

Comment:
I have to check how does it look, because otherwise I would try to use this: https://devcenter.heroku.com/articles/app-json-schema#scripts